### ISSUE TITLE
This plugin add excessive Run option

### ISSUE THREAD

**Issue by paulvi** [angelozerr/jsbuild-eclipse#7]
This plugin add excessive Run option 

e.g. for project without JS or Tern nature, for xml, gradle files, any folder etc.

Filters should be applied.

**Comment by angelozerr** [comment]
Yes you have right. I had done somthing but it seems that there are some bugs. I will fix when I will find time. Thanks for the issue.

**Comment by angelozerr** [comment]
@paulvi I will fix soon this issue. It's because propertiesTester is not loaded. If you open the build explorer view (which force teh load of grunt/gulp plugins), your problem should disappear
